src/jshint.js

```diff
@@ -2880,7 +2880,7 @@ var JSHINT = (function() {
 
     if (loneArg && loneArg.identifier === true) {
```

Comment:
Unrelated, but we should really, really make it clear that `loneArg` is a token --- maybe rename it to clarify that